### PR TITLE
Updating api/validation/schema_test to test testapi.Version rather than just v1beta1

### PR DESCRIPTION
**Pull request by nikhiljindal**
For https://github.com/GoogleCloudPlatform/kubernetes/issues/6584.

As part of the change, updated api/validation/schema to use the spec from api/swagger-spec.

### CODE REVIEW

**Comment by nikhiljindal** [comment]
@bgrant0607

File: cmd/kube-version-change/version.go [hunk 1]
```diff
@@ -78,7 +78,14 @@ func main() {
 	if err != nil {
 		log.Fatalf("Couldn't read from input: %q", err)
 	}
+	isYAML := isYAML(data)
 
+	if isYAML {
```
Comment (bgrant0607) [inline cmd/kube-version-change/version.go]: Thanks.

Without a test, this is probably going to be broken again.

@lavalamp

**Comment by bgrant0607** [comment]
LGTM

File: cmd/kube-version-change/version.go (hunk 1 above)
Comment (lavalamp) [inline cmd/kube-version-change/version.go]: Thanks for fixing, I agree this should probably get a package and a test.

**Pull request merged**